src/main/java/com/elixr/poc/rest/response/CommonResponse.java

```diff
@@ -1,9 +1,11 @@
 package com.elixr.poc.rest.response;
 
+import lombok.Builder;
 import lombok.Data;
 
 @Data
-public class DeleteResponse {
+@Builder
+public class CommonResponse {
```

Comment:
What is this response used for? You already have another common response class?

Reply:
Made changes.